pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[project]
+name = "soar-sml"
+description = "Raw SML Bindings to the SOAR Cognitive Architecture"
+version = "9.6.2"
+license = "BSD"
+authors = [
+    # {name = "Author Name", email = "authoremail@example.com"}
+]
+readme = "README.md"
+# Enscons does not support 3.12 as of yet
+requires-python = ">=3.8,<3.12"
```

Comment:
(Cibuildwheel does that machinery c:)

As I said in a below comment: I'm thinking about forking enscons to add small fixes to problems I've noticed along the way, and to fix the 3.12 incompatibility, plus others

Reply:
Wow, okay cibuildwheel is doing lot more than I thought. And this would resolve another important pain I've seen coming for a while (want the new Soar? Gotta upgrade your Python!).

Reply:
Yes, it currently builds for pythons 3.8 through 3.11; You can install `soar-sml` for any one of them via this method :)

Reply:
I've fixed support for python 3.12 by using a fork of enscons